src/App.tsx

```diff
@@ -1,39 +1,50 @@
-import { useState } from 'react'
-import { Routes, Route, useLocation } from 'react-router-dom'
-import { Navbar } from './components/navbar/Navbar'
-import { Header } from './components/header/header'
-import { CreateClass } from './components/modal/create-class'
-import { Button } from './components/button/Button'
-import './App.css'
+import { useState } from "react";
```

Comment:
To com a impressão de que esse arquivo inteiro é mudança de formatação. Não sei se é bom ficar mudando formatação toda hora, além disso, dá uma confundida pra quem revisa